src/Mvc/Mvc.Core/src/MvcCoreLoggerExtensions.cs

```diff
@@ -72,7 +72,7 @@ internal static class MvcCoreLoggerExtensions
 
         private static readonly Action<ILogger, string, Exception> _localRedirectResultExecuting;
 
-        private static readonly Action<ILogger, string, Exception> _objectResultExecuting;
+        private static readonly Action<ILogger, ObjectResult, string, Exception> _objectResultExecuting;
```

Comment:
One small change we should make to this:

Make this a string rather than `ObjectResult`, and have `ObjectResultExecuting` do `result.GetType().Name`. This code currently relies on the expectation that `ObjectResult.ToString()` is going to print the type name (the default). 

If someone overrode `.ToString()` it would change what's logged to not match expectations. Imagine someone overrode `.ToString()` to include `.Value.ToString()` 😈. We explicitly go out of our way to log the type name of the value, not the value itself, users are often concerned about this from a security POV.

Reply:
I also think it would be nice to use `.GetType().Name` here instead of `.GetType().FullName`. Most object results are defined by us, so printing the namespace is pretty repetative, and would be less readable than what it does today.

Reply:
I implemented it like that because that is how it works for `FileResult` a few lines above (line 57, 58). I thought to copy the solution that there already was.

Would it make sense to change them all to use `.GetType().Name` or should I just stick to `ObjectResult`?

Reply:
I went ahead and made the change to `FileResult` logging too. Feel free to ask me to revert.